cli/EAP7-1113_Disable_CLI_paging_with_an_option.adoc

```diff
@@ -0,0 +1,66 @@
+= EAP7-1113 Disable CLI paging with an option
+:author:            Marek Marusic
+:email:             mmarusic@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          cli,disable,option,output,scrolling
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The output paging is not desired in some cases e.g. when copy/pasting the full output of a command.
+The paging could be disabled by a simple option.
+
+=== Workaround
+It is currently possible to override the paging of a single command by redirecting the output to /dev/stdout.
+
+==== Example of the workaround:
+----
+/subsystem=elytron:read-resource-description(recursive=true) > /dev/stdout
+----
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1113[EAP7-1113]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFCORE-4127[WFCORE-4127]
+* https://issues.jboss.org/browse/WFCORE-4096[WFCORE-4096]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts
+
+* mailto:<email>@redhat.com[TBD]
+
+=== Affected Projects or Components
+
+* CLI
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+* Ability to disable output paging via CLI argument (--no-output-paging)
```

Comment:
Here should be provided example of CLI command where this option will be used. 

--no-output-paging does not seem to be intuitive. At the first look i'm not sure what paging means. What about some other name? Maybe  --no-output-scrolling wdyt?

Reply:
I vote for "paging" :)

Reply:
I would vote for --no-paging, paging seems the right word.

Reply:
Ok, let's have "--no-paging"

Reply:
Thank you everyone I have made the requested changes.